src/sql/src/plan/statement/ddl.rs

```diff
@@ -712,8 +712,21 @@ pub fn plan_create_view(
     } else {
         scx.allocate_name(normalize::object_name(name.to_owned())?)
     };
+    let (mut relation_expr, mut desc, finishing) =
+        query::plan_root_query(scx, query.clone(), QueryLifetime::Static)?;
+    relation_expr.bind_parameters(&params)?;
+    //TODO: materialize#724 - persist finishing information with the view?
+    relation_expr.finish(finishing);
+    let relation_expr = relation_expr.decorrelate();
```

Comment:
Could `let relation_expr = relation_expr.decorrelate();` be moved after the `replace` check? I don't know if it impacts the `global_uses`, but if not, it'll give the user an error a fraction of a second more quickly.

Reply:
Yeah, `global_uses` is only a thing on `::expr::RelationExpr`s.